### DISCUSSION TITLE
Mycroft OS stuck at booting

### DISCUSSION THREAD

**Discussion by JairoJosy** [OpenVoiceOS/ovos-buildroot#72] [Q&A] [answered]
Hi, I recently downloaded the Mycroft OS for Raspberry Pi 3, flashed it onto to a fresh sd card and booted. As I was using ethernet it didnt require me for wifi setup and it directly booted into a bluescreen with Mycroft AI written. I waited for 1 hour and after that said"Hey Microft, What is the time" and It responded like "Please wait a moment as I finish booting up". So, I waited for another 1 hour and still the same. Is there any way I get it running. It is very urgent to me. Pls help!!!!!!!

Technical Specifications :
Hardware : Raspberry Pi 3B+
OS - Mycroft OS download from : https://github.com/OpenVoiceOS/OpenVoiceOS/releases/download/0.1.0_alpha7/MycroftOS_0.1_rpi3_alpha7.img

**Comment by ChanceNCounter** [comment, marked as answer]
Thread addressed in chat, so I'm closing it here.

For the future curious: the image linked above shouldn't be up anymore, so I've taken it down. As of this date, we don't promise the prealpha repo will boot on Pi3. Pi3 compatibility *will be restored*, but, since it takes many hours to build an image, we're focusing on what is currently the more popular hardware target.
